src/Button/index.js

```diff
@@ -110,4 +110,8 @@ Button.defaultProps = {
   type: 'button',
 };
 
-export default styled(Button)``;
+const styledButton = styled(Button)``;
```

Comment:
I don't understand here. Why don't you do : 
```jsx
Button.displayName = 'Button'

export default styled(Button)``;
```

Reply:
@AntoninMarchardDev Because of styled-component! If I Iexport like this `export default styled(Button)``;` React displayName is styled.Button

Reply:
I didn't know !